File: tests/scripts/subcommands/compile_and_run/stage-with-grader/public/java/grader.java
import java.util.Scanner;

public class grader {
	public static void main(String... args) {
		Scanner sc = new Scanner(System.in);
		int a = sc.nextInt();
		int b = sc.nextInt();
		sc.close();

		add1 solver = new add1();
		int res = solver.solve(a, b);

		try (java.io.PrintWriter out = new java.io.PrintWriter(System.out)) {
			out.println(res);
		}
	}
}

File: tests/scripts/subcommands/compile_and_run/stage-with-grader/public/cpp/grader.cpp
#include "add1.h"
#include <cassert>
#include <cstdio>

int main() {
	int a, b;
	assert(2 == scanf("%d%d", &a, &b));
	fclose(stdin);

	int res = solve(a, b);

	printf("%d\n", res);
	fclose(stdout);
	return 0;
}
